app-eselect/eselect-mpg123/eselect-mpg123-0.1-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Manage /usr/bin/mpg123 symlink"
+HOMEPAGE="https://www.gentoo.org/proj/en/eselect/"
+SRC_URI=""
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
+IUSE=""
```

Comment:
Drop definition, this is okay per [PMS](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-620007.3)

Reply:
Done